db/seeds.rb

```diff
@@ -1,42 +1,21 @@
-YAML.load_file('db/team.yml')['team'].each do |hash|
-  params = hash.symbolize_keys
-  introduction = params.delete(:introduction)
-  person = Person.create params
+require 'oj'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.